test/integration/containerd_test.go

```diff
@@ -0,0 +1,82 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/aquasecurity/fanal/image/daemon"
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/stretchr/testify/require"
+	testcontainers "github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	redisImageName  = "index.docker.io/library/redis:latest"
+	mysqlImageName  = "index.docker.io/library/mysql:latest"
+	pythonImageName = "index.docker.io/library/python:3.4-alpine"
```

Comment:
Can we import tarballs into containerd?
https://github.com/aquasecurity/fanal/blob/d15bba5c3713716a99826c8500a260536f322fad/Makefile#L19

That is what we do for Docker Engine.
https://github.com/aquasecurity/fanal/blob/d15bba5c3713716a99826c8500a260536f322fad/test/integration/library_test.go#L171-L174